lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -173,9 +173,13 @@
     }
   }
 
+  .ccx-grade-book-download {
+    text-align: right;
+  }
+
   h1 {
     @extend .top-header;
-    display: inline-block;
+    display: block;
```

Comment:
It seems like this has the potential for serious side effects. Is there some other way we can do it to limit the scope? Is this a `h1` that appears *inside* the embedded gradebook?